src/Engine/Utils.java

```diff
@@ -501,6 +501,63 @@ public static void sortLocationsByTravelTime(Unit unit, ArrayList<XYCoord> mapLo
     TravelDistanceComparator tdc = new TravelDistanceComparator(unit, map);
     Collections.sort(mapLocations, tdc);
   }
+  
+
+  /** Returns a string describing the game statistics of a given CO */
+  public static String getGameStatusData(GameMap map, Commander viewed)
```

Comment:
This feels like it should probably belong in `GameInstance`. Unless there's a reason it should go here instead?

Reply:
Uh, not particularly? AFAIK, it's not much different from figuring out where a unit can move.
Why do you think it should go in GameInstance?

Reply:
It's asking for a bunch of game-specific data. I guess it's the difference between asking GameInstance to tell about itself vs asking Utils to tell us about GameInstance. Utils stuff is generally functional; figuring out what can be done or how to do it, and this feels slightly out of place since it's just providing statistics. Maybe UIUtils? Though that's more about factions and colors right now. I dunno, I'm not sure how much it matters where it goes, really.